src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs

```diff
@@ -132,6 +137,16 @@ public Task StartAsync(Uri url, Channel<byte[], SendMessage> application, Transf
                 _transportCts.Cancel();
                 stream.Dispose();
                 _logger.ReceiveStopped(_connectionId);
+                pipe.Writer.Complete();
+
+                try
+                {
+                    await writeTask;
```

Comment:
Should we handle cases where this task takes too long/never finishes, or is that not something that happens except in test code